.github/workflows/build-lint-test.yml

```diff
@@ -31,10 +40,12 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: '18'
+          node-version: 'lts/*'
           cache: 'pnpm'
 
       - run: pnpm install
+      - run: pnpm install
+        working-directory: docker
```

Comment:
install any additional dependencies in the `docker/` folder to prevent linting or type checking errors in this folder